tests/test_apps/app_with_callbacks/default_callback/hydra_plugins/env_plugin/env_searchpath_plugin.py

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+from hydra.core.config_search_path import ConfigSearchPath
+from hydra.plugins.search_path_plugin import SearchPathPlugin
+
+
+class ExampleSearchPathPlugin(SearchPathPlugin):
+    def manipulate_search_path(self, search_path: ConfigSearchPath) -> None:
+        search_path.prepend(
+            provider="test-env-defaults",
+            path="pkg://hydra_plugins.env_plugin.conf",
+            anchor="hydra",
+        )
```

Comment:
What is an Env Plugin?

Let's pull out the env related stuff to a separate PR. It deserves proper review.